drivers/modem/Kconfig.gsm

```diff
@@ -78,6 +78,20 @@ config MODEM_GSM_MANUAL_MCCMNO
 	  the network provider and may need to be changed if auto is not
 	  selected.
 
+config MODEM_GSM_RSSI_WORK_PERIOD
+	int "Configure RSSI WORK polling period"
```

Comment:
Please add some info about the time resolution.
So something like `Configure RSSI polling period (in seconds)`,
also the "WORK" does not need to be mentioned here as it does not give any useful info.

Reply:
thanks, addressed now